From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jaime Dias <jaimejosediasnt@gmail.com>
Date: Mon, 3 Aug 2026 18:59:32 -0300
Subject: [PATCH 05/11] DEBUG: turn on tracing in dos, lddemon and Shell

Bring-up aid, not part of the port. Enables the existing D() tracing in
the boot and shell paths, which is where a new architecture spends most
of its early failures.

Applied last and touching nothing else, so it can be dropped without
disturbing the rest of the series.
---
 rom/dos/boot.c            | 1 +
 rom/dos/cliinit.c         | 1 +
 rom/dos/newcliproc.c      | 1 +
 rom/dos/shell_helper.c    | 1 +
 rom/dos/systemtaglist.c   | 1 +
 rom/lddemon/lddemon.c     | 1 +
 workbench/c/Shell/Shell.c | 1 +
 7 files changed, 7 insertions(+)

diff --git a/rom/dos/boot.c b/rom/dos/boot.c
index adf0a3684a..8076faf602 100644
--- a/rom/dos/boot.c
+++ b/rom/dos/boot.c
@@ -4,6 +4,7 @@
     Desc: Implements AROS's generic/amiga-like boot sequence.
 */
 
+#define DEBUG 1
 #include <aros/debug.h>
 #include <exec/alerts.h>
 #include <exec/libraries.h>
diff --git a/rom/dos/cliinit.c b/rom/dos/cliinit.c
index 218cb9a579..4efec393b2 100644
--- a/rom/dos/cliinit.c
+++ b/rom/dos/cliinit.c
@@ -4,6 +4,7 @@
     Desc:
 */
 
+#define DEBUG 1
 #include <aros/debug.h>
 #include <exec/resident.h>
 #include <proto/exec.h>
diff --git a/rom/dos/newcliproc.c b/rom/dos/newcliproc.c
index f024bb2cea..8c605126c9 100644
--- a/rom/dos/newcliproc.c
+++ b/rom/dos/newcliproc.c
@@ -4,6 +4,7 @@
     Desc:
 */
 
+#define DEBUG 1
 #include <aros/debug.h>
 
 #include <dos/dos.h>
diff --git a/rom/dos/shell_helper.c b/rom/dos/shell_helper.c
index b39987d288..6f54d924e7 100644
--- a/rom/dos/shell_helper.c
+++ b/rom/dos/shell_helper.c
@@ -4,6 +4,7 @@
     Desc:
 */
 
+#define DEBUG 1
 #include <aros/debug.h>
 #include <utility/tagitem.h>
 #include <dos/dostags.h>
diff --git a/rom/dos/systemtaglist.c b/rom/dos/systemtaglist.c
index 7a5f675e4a..463b08b53c 100644
--- a/rom/dos/systemtaglist.c
+++ b/rom/dos/systemtaglist.c
@@ -4,6 +4,7 @@
     Desc:
 */
 
+#define DEBUG 1
 #include <aros/debug.h>
 #include <utility/tagitem.h>
 #include <dos/dostags.h>
diff --git a/rom/lddemon/lddemon.c b/rom/lddemon/lddemon.c
index 58d8e6d2bb..03b276ffcd 100644
--- a/rom/lddemon/lddemon.c
+++ b/rom/lddemon/lddemon.c
@@ -6,6 +6,7 @@
 
 #include <aros/config.h>
 
+#define DEBUG 1
 #include <aros/asmcall.h>
 #include <aros/debug.h>
 #include <aros/symbolsets.h>
diff --git a/workbench/c/Shell/Shell.c b/workbench/c/Shell/Shell.c
index bf13ddcd6e..0c27a3a386 100644
--- a/workbench/c/Shell/Shell.c
+++ b/workbench/c/Shell/Shell.c
@@ -8,6 +8,7 @@
 
 #define __EXEC_NOLIBBASE__
 
+#define DEBUG 1
 #include <aros/debug.h>
 
 #include <dos/dos.h>
